docs/docusaurus/docs/gx_cloud/tutorials/getting_started/getting_started_with_gx_cloud.md

```diff
@@ -177,18 +152,15 @@ Now that we have connected to data and defined an <TechnicalTag tag="expectation
 ```python title="Jupyter Notebook"
 checkpoint_name = None # name your checkpoint here
 assert checkpoint_name is not None, "Please set checkpoint_name."
-data_asset_name = None # name your table here
-assert data_asset_name is not None, "Please set data_asset_name."
 
 checkpoint_config = {
   "name": checkpoint_name,
   "validations": [{
       "expectation_suite_name": expectation_suite_name,
       "expectation_suite_ge_cloud_id": expectation_suite.ge_cloud_id,
       "batch_request": {
```

Comment:
I believe this still works if you omit `expectation_suite_ge_cloud_id` - maybe worth investigating but will not block on it.

Reply:
This works as is.
I tried many different iterations that don't include a `batch_request` as part of the checkpoint config and instead pass it to `checkpoint.run()` but I couldn't get anything to work.
```
StoreBackendError: Unable to set object in GX Cloud Store Backend: [{"detail": "Substituted Validation with idx 0 is invalid: validation batch_request cannot be None.", "source": {"pointer": "/data/attributes/checkpoint_config/validations[0]"}}]
```